source/modulo_new_core/test/cpp_tests/translators/test_translators.cpp

```diff
@@ -0,0 +1,175 @@
+#include <gtest/gtest.h>
+
+#include "modulo_new_core/translators/ReadStateConversion.hpp"
+#include "modulo_new_core/translators/WriteStateConversion.hpp"
+
+#include <rclcpp/clock.hpp>
+
+using namespace modulo_new_core::translators;
+
+static void expect_vector_equal(const Eigen::Vector3d& v1, const geometry_msgs::msg::Vector3& v2) {
+  EXPECT_FLOAT_EQ(v1.x(), v2.x);
+  EXPECT_FLOAT_EQ(v1.y(), v2.y);
+  EXPECT_FLOAT_EQ(v1.z(), v2.z);
+}
+
+static void expect_point_equal(const Eigen::Vector3d& vector, const geometry_msgs::msg::Point& point) {
+  EXPECT_FLOAT_EQ(vector.x(), point.x);
+  EXPECT_FLOAT_EQ(vector.y(), point.y);
+  EXPECT_FLOAT_EQ(vector.z(), point.z);
+}
+
+static void expect_quaternion_equal(const Eigen::Quaterniond& q1, const geometry_msgs::msg::Quaternion& q2) {
+  EXPECT_FLOAT_EQ(q1.x(), q2.x);
+  EXPECT_FLOAT_EQ(q1.y(), q2.y);
+  EXPECT_FLOAT_EQ(q1.z(), q2.z);
+  EXPECT_FLOAT_EQ(q1.w(), q2.w);
+}
+
+class TestAicaMsgs : public ::testing::Test {
```

Comment:
I see you previously had a different name for this package 😄 By convention, test fixture classes should be named `FooTest`. I would suggest something along the lines of:
```suggestion
class TranslatorTest : public ::testing::Test {
```

Reply:
hehe you caught me